Adding issue templates (#247)
meilisearch/meilisearch-python: .github/ISSUE_TEMPLATE/feature_request.md

---
name: Feature Request 💡
about: Suggest a new idea for the project.
title: ''
labels: enhancement
assignees: ''
---

> [comment]: <> (If the feature happens in multiple SDK, open it in the integration-guide instead: https://github.com/meilisearch/integration-guides/issues)

Reviewer: ```suggestion
[comment]: <> (If the feature deserves to be added into multiple SDKs, open an issue in the [integration-guide](https://github.com/meilisearch/integration-guides/issues) instead.)
```

> [comment]: <> (This is not an exhaustive model but a help. Nothing is mandatory)

Reviewer: ```suggestion
[comment]: <> (This template is not an exhaustive model but a help. No step is mandatory.)
```

> [comment]: <> (This is not an exhaustive model but an help. Nothing is mandatory)

Reviewer 2: ```suggestion
[comment]: <> (This is not an exhaustive model but a help. Nothing is mandatory)
```

**Description**
Brief explanation of the feature.

**Basic example**

> If the proposal involves something new or a change, include a basic example. Omit this section if it's not applicable.

Reviewer: ```suggestion
If the proposal involves something new or a change, include a basic example.
```

Reviewer: No need to say it's not mandatory since you already say "If the proposal involves something new or a change"

> If the proposal involves something new or a change, include a basic example. Omit this section if it's not applicable.

Reviewer: ```suggestion
If the proposal involves something new or a change, include a basic example. How do would you use the feature? In which context?
```

**Additional context**

> Add any other context about the feature here.

Reviewer: I don't know if this step is really useful. For me, all the description will be done in the paragraphs above

Author: I am shared between not to propose too much and also left space for free expression.
We can change it for:
**Other**
Any other things you want to add.
For now, I remove these paragraphs!

Reviewer: I link this "Other", it's more friendly :)

Author: Done
